src/encoded/schemas/genetic_modification.json

```diff
@@ -361,7 +359,7 @@
                 {
                     "properties": {
                         "category": {
-                            "enum": ["deletion", "interference"]
+                            "enum": ["CRISPR cutting", "interference", "CRISPRi"]
```

Comment:
would not it make sense to break into 2: one for CRISPR (then it is CRISPRi or CRISPRcutting) and one for RNAi where it is ONLY interference?